.github/workflows/unit.yml

```diff
@@ -2,6 +2,12 @@ name: unit
 
 on:
   workflow_call:
+    inputs:
```

Comment:
Shouldn't it be workflow-level? For `worflow_dispatch` and `schedule` this won't work as far as I can understand.

Reply:
Our goal here is to make those workflows reusable and then later on reuse them in other workflows that are getiting triggered manualy by schedule or any other way.
`workflow_call` makes workflow reusable and can be called with custom input.
[Example from my other branch ](https://github.com/neptune-ai/neptune-client/blob/4438f30304d72756fb59094e78d1d37ffd48499d/.github/workflows/e2e.yml#L174)

Reply:
So let's remove the `schedule` and `workflow_dispatch` from `dev/1.x` then. This workflow will only be triggered externally or via a `push`.

This will be broken for `push` to `dev/1.x` as `inputs.neptune_ref` will not be provided as far as I can undestand.

Reply:
> This will be broken for push to `dev/1.x` as `inputs.neptune_ref` will not be provided as far as I can undestand.
For now `neptune_ref` defaults to `${{ github.event.client_payload.pull_request.head.ref }}` if not provided but I guess it will brake on push

Reply:
I think that this will not be set for `push` as the `inputs` default was added under "workflow_call". Shouldn't we somehow add a global inputs settings or use some inputs getter with a default?

Reply:
Yes I think that is great idea but I'd argue its out of scope for this task